ui/raidboss/data/07-dt/raid/r8s.ts

```diff
@@ -1,14 +1,1907 @@
+        })).combatants;
+        const actor = actors[0];
+        if (actors.length !== 1 || actor === undefined) {
+          console.error(
+            `R8S Eminent/Revolutionary Reign Direction: Wrong actor count ${actors.length}`,
+          );
+          return;
+        }
+
+        switch (matches.id) {
+          case eminentReign1:
+          case eminentReign2:
+            data.reignDir = (Directions.hdgTo16DirNum(actor.Heading) + 8) % 16;
+            break;
+          case revolutionaryReign1:
+          case revolutionaryReign2:
+            data.reignDir = Directions.hdgTo16DirNum(actor.Heading);
+            break;
+        }
+      },
+      infoText: (data, matches, output) => {
+        const dir = output[Directions.output16Dir[data.reignDir ?? -1] ?? 'unknown']!();
+        switch (matches.id) {
+          case eminentReign1:
+          case eminentReign2:
+            return output.inDir!({ dir: dir });
+          case revolutionaryReign1:
+          case revolutionaryReign2:
+            return output.outDir!({ dir: dir });
+        }
+      },
+      run: (data) => {
+        data.reignDir = undefined;
+      },
+      outputStrings: {
+        ...Directions.outputStrings16Dir,
+        inDir: {
+          en: 'In ${dir}',
+        },
+        outDir: {
+          en: 'Out ${dir}',
+        },
+      },
+    },
+    {
+      id: 'R8S Millenial Decay',
+      type: 'StartsUsing',
+      netRegex: { id: 'A3B2', source: 'Howling Blade', capture: false },
+      response: Responses.bigAoe(),
+    },
+    {
+      id: 'R8S Breath of Decay Rotation',
+      type: 'StartsUsing',
+      netRegex: { id: 'A3B4', source: 'Wolf of Wind', capture: true },
+      durationSeconds: 6,
+      infoText: (data, matches, output) => {
+        // 1st add always spawns N or S, and 2nd add always spawns intercardinal
+        // we only need the position of the 2nd add to determine rotation
+        data.decayAddCount++;
+        if (data.decayAddCount !== 2)
+          return;
+
+        const addX = parseFloat(matches.x);
+        const addY = parseFloat(matches.y);
+        const addDir = Directions.xyTo8DirNum(addX, addY, centerX, centerY);
+        if (addDir === 1 || addDir === 5)
+          return output.clockwise!();
+        else if (addDir === 3 || addDir === 7)
+          return output.counterclockwise!();
+      },
+      outputStrings: {
+        clockwise: {
+          en: '<== Clockwise',
+        },
+        counterclockwise: {
+          en: 'Counterclockwise ==>',
+        },
+      },
+    },
+    {
+      id: 'R8S Aero III',
+      // Happens twice, but Prowling Gale occurs simultaneously on the second one
+      type: 'StartsUsing',
+      netRegex: { id: 'A3B7', source: 'Howling Blade', capture: false },
+      suppressSeconds: 16,
+      response: Responses.knockback(),
+    },
+    {
+      id: 'R8S Prowling Gale Tower/Tether',
+      // Calls each tether or get towers
+      // TODO: Support getting a tower and tether?
```

Comment:
I don't think this TODO is viable. Even if somehow a tank realized they had to adjust to take a tower and tether (and then invuln to survive), and even if they managed that adjust, the cleave from their tether will always hit another player, I believe.

Reply:
Early on I believe there were starts involving just tether players taking the towers. It's doable without cleaving others, and easier when the tower is missing a player altogether. But the ask is a bit much.

Reply:
I have removed the comment.